core/shared/src/main/scala/caseapp/core/complete/Bash.scala

```diff
@@ -19,7 +21,7 @@ object Bash {
   }
 
   private def escape(s: String): String =
-    s.replace("\"", "\\\"")
+    s.replace("\"", "\\\"").replace("`", "\\`").split(Properties.lineSeparator).headOption.getOrElse("")
```

Comment:
I'm not sure, but I fear the line separator in the string is going to be the one of the OS the case-app user code is compiled on, not the one from the end user OS (that `Properties.lineSeparator` gives). But something like [this](https://stackoverflow.com/a/454913/3714539) should work (by being OS-agnostic).